### DISCUSSION TITLE
Issue with Importing Printer Profiles into Prusaslicer: Printer Notes Field too Long

### DISCUSSION THREAD

**Discussion by MartinLunn** [sn4k3/UVtools#185] [Q&A] [answered]
I might be making a silly mistake, but I ran into an issue while following the instructions here: https://github.com/sn4k3/UVtools/wiki/Setup-PrusaSlicer 

Specifically instruction 6. iv.

It's telling me that I can't save the printer notes because they're too long. I think I changed my light off delay, from a 0 to a 9 or something, so it's the same length.

Assistance would be appreciated!

![image](https://user-images.githubusercontent.com/10377451/115830483-59d30180-a3de-11eb-9694-ca6865ecc2ad.png)

If I try and telling to save anyway, this is what the notes say:
![image](https://user-images.githubusercontent.com/10377451/115830906-e54c9280-a3de-11eb-952b-c44150b98e43.png)

**Comment by sn4k3** [comment, marked as answer]
Where is telling that you can't save because too long?
There is some problem there with profile, it says Ender3 so you are using it as FDM printer... Notes are all wrong, printer too.
Make sure you have imported SL1 printer from wizard, start my tutorial from start again

**Reply by MartinLunn** [reply]
Thanks for all the help! 
Too much new software at once is a recipe for stupid mistakes.
